extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLogController.class.php

```diff
@@ -207,7 +214,7 @@ private function aggregateLogByIpAddress( $ipAddress ) {
 	private function aggregateSearchLogs( $query ) {
 		$records = [];
 		$dayOffset = 0;
-		$ipHash = [];
```

Comment:
Identical ip/domain tuples should still be eliminated, right?
Also come to think of it, should this become an ip/domain/userId tuple or else will it eliminate users to the same site within the same ip?